spec.html

```diff
@@ -10971,17 +11445,31 @@ <h1>MakeMethod ( _F_, _homeObject_ )</h1>
       </emu-alg>
     </emu-clause>
 
+    <emu-clause id="sec-definemethodproperty" aoid="DefineMethodProperty">
+      <h1>DefineMethodProperty ( _key_, _homeObject_, _closure_, _enumerable_ )</h1>
+      <p>The abstract operation DefineMethodProperty takes arguments _key_ (a property key or Private Name), _homeObject_ (an Object), _closure_ (a function object), and _enumerable_ (a Boolean). It performs the following steps when called:</p>
+      <emu-alg>
+        1. Perform SetFunctionName(_closure_, _key_).
```

Comment:
I've added it here, but the prefix is not strictly necessary according to the spec's current rules (since it's just `Perform`ing, not consuming the value), so I'd prefer not to change the existing sites now and instead just fix them up as part of https://github.com/tc39/ecma262/issues/1796.

Reply:
It'd be necessary if it returned abruptly, no?

Reply:
`SetFunctionName` cannot return abruptly.

Reply:
sure, but i can't know that at the callsite (pending #1796)